composer.json

```diff
@@ -39,7 +39,8 @@
 		}
 	],
 	"require": {
-		"php": ">=5.3.3",
+		"php": ">=5.4",
+		"phpbb/phpbb": ">=3.2.0@dev",
```

Comment:
@michaelcullum we disable packagist but that's not an issue: phpbb/phpbb is available in the current installed version as for all phpBB installed dependencies. And any other deps required by the extensions could be proxoyfied from Titania or any other available repo.

About the version, we now 3.3, if it exists, will break something. So I'll set it to `~3.2.0@dev`